example_site/src/components/Header.css

```diff
@@ -1,5 +1,11 @@
+.center {
```

Comment:
Typically a classname like this is something that's shared among a lot of different elements because it's just conveying that they should be centered.

Since this class specifically applies to the header, let's give it a name that ties it to the component, like `HeaderContainer`

Reply:
This doesn't matter since we're getting rid of the `center` class in `App.css`, but remember that everything inherits from `App`. You see how you can use the `form-input` class in any component even though it's defined in `App.css`? The same could have been said for the `center` class so it's a good idea to avoid class name collisions like that.